src/zope/sendmail/queue.py

```diff
@@ -362,39 +362,56 @@ class ConsoleApp(object):
-                           "messages in queue once and exit.")
-    parser.add_option('--interval', metavar='<#secs>', type='float', default=3,
-                      help="How often to check queue when in daemon mode. "
-                           "Default is %default seconds.")
-    parser.add_option('--hostname', default='localhost',
-                      help="Name of smtp host to use for delivery.  Default is "
-                           "%default.")
-    parser.add_option('--port', type='int', default=25,
-                      help="Which port on smtp server to deliver mail to. "
-                           "Default is %default.")
-    parser.add_option('--username',
-                      help="Username to use to log in to smtp server.  Default "
-                           "is none.")
-    parser.add_option('--password',
-                      help="Password to use to log in to smtp server.  Must be "
-                           "specified if username is specified.")
-    parser.add_option('--force-tls', action='store_true',
-                      help="Do not connect if TLS is not available.  Not "
-                           "enabled by default.")
-    parser.add_option('--no-tls', action='store_true',
-                      help="Do not use TLS even if is available.  Not enabled "
-                           "by default.")
-    parser.add_option('--config', metavar='<inifile>',
-                      help="Get configuration from specified ini file; it must "
-                           "contain a section [%s] that can contain the "
-                           "following keys: %s. If you specify the queue path "
-                           "in the ini file, you don't need to specify it on "
-                           "the command line." % (INI_SECTION,
-                                                  ', '.join(INI_NAMES)))
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--daemon', action='store_true',
+                        help=("Run in daemon mode, periodically checking queue "
+                              "and sending messages.  Default is to send all "
+                              "messages in queue once and exit."))
+    parser.add_argument('--interval', metavar='<#secs>', type=float, default=3,
+                        help=("How often to check queue when in daemon mode. "
+                              "Default is %(default)s seconds."))
+    smtp_group = parser.add_argument_group("SMTP Server",
+                                           "Connection information for the SMTP server")
+    smtp_group.add_argument('--hostname', default='localhost',
+                            help=("Name of smtp host to use for delivery.  Default is "
+                                  "%(default)s."))
+    smtp_group.add_argument('--port', type=int, default=25,
+                            help=("Which port on smtp server to deliver mail to. "
```

Comment:
I would correct the capitalization of SMTP in both of the above help messages.

Reply:
OK.